### ISSUE TITLE
Windows SDK Version

### ISSUE THREAD

**Issue by jcs100** [hexchat/gtk-win32#37]
Thank you for your work building up to date libraries for Win32.

Are the libraries compatible with Windows XP? i.e. were they built with the v140_xp platform toolset?

**Comment by TingPing** [comment]
No, HexChat doesn't target XP. Also I believe GLib and a few other deps already dropped XP support too.

**Comment by jcs100** [comment]
Thanks for the information. I will try to build the project with the 140_xp platform toolset and see what happens.

**Comment by TingPing** [comment]
I might suggest the wingtk project to you: https://github.com/wingtk/gtk-win32

It is a fork of this repo that intends to be more useful for the general public compared to this which is HexChat specific. I don't know if they would want to actually support XP either but you could have a conversation over it and it might be a better base anyway.
